.github/workflows/ci.yml

```diff
@@ -45,12 +76,34 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Cache sbt
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.cache/coursier
+          ~/.sbt
+        key: ${{ runner.os }}-${{matrix.scala}}-${{matrix.module}}-${{ hashFiles('**/build.sbt') }}
 
     - name: Setup Java and Scala
       uses: olafurpg/setup-scala@v10
       with:
         java-version: 1.8.0
 
-    - run: export TRAVIS_SCALA_VERSION=${{ matrix.scala }}
-    - run: export TRAVIS_BRANCH=$(git for-each-ref ${{ github.ref }} --format='%(refname:short)')
-    - run: ./build/release.sh ${{ matrix.scala_short }} ${{ matrix.module }}
+    - name: Release
```

Comment:
Not sure if it's beyond the scope of this commit but we need to test that releases work. In ProtoQuill I got a github-actions release to work by changing the keys and and other stuff:
https://github.com/getquill/protoquill/blob/master/build/release.sh